+++
title = "Riichi"
arcturus_wiki_id = "209"
tags = ["Yaku", "Game rules"]
+++

```yaku
|type = Yaku
|kanji = 立直 or リーチ
|english = Ready hand
|value = 1 han (closed only)
|yakuSpeed = Varies
|difficulty = At discretion
|yakuCombine = \* All standard yaku

  - May be used with yakuman, but does not affect score

|gameExample = \* [With ippatsu](http://tenhou.net/0/?log=2013080615gm-0009-7447-xa503715f0f66&tw=3&ts=5)

  - [Three riichis called](http://tenhou.net/0/?log=2013053111gm-0009-0000-d06067c3&tw=1&ts=4)
  - Ippatsu oikake riichi"
  - [Double riichi](http://tenhou.net/0/?log=2012120515gm-0009-7447-637daef9&tw=3&ts=3)

```

**Riichi** {{< kana "立直" >}} or {{< kana "リーチ" >}} is the most commonly used yaku in the game.
Among the yaku, riichi occurs at least 40% of the time across different mahjong platforms and
professional settings. This yaku may apply to any closed hand and may be declared upon achieving
tenpai. As such, riichi is attainable via numerous hand configurations, as long as it is closed. Due
to its common usage, various specific game scenarios must be considered when using riichi, or facing
opponents who have called riichi.

Unlike other yaku, the call for riichi operates at a player's discretion and is certainly not
mandatory. Players may simply opt not to call riichi and retain the secrecy of a hand's tenpai
status, using a tactic known as damaten. The decision for using riichi rests on some factors,
primarily on need or desire. Current point standings often factor towards a riichi call, especially
when falling behind in points. On the other hand, point leaders may tend to be conservative to limit
opportunities for other players to score hands directly.

## Rules

When a player has a closed tenpai hand, the player may declare riichi. Doing so gains the hand 1
han. To declare riichi, a player announces riichi and discards a tile facing sideways in the discard
pile. This is done to indicate when riichi was called. If that tile is claimed by another player for
an open meld, then the next discard is turned sideways as a replacement.

Unless the first sideways discard is claimed for a win immediately, the riichi announcer now places
a bet of 1,000 points on the table. This bet is collected by the next player to win a hand. Of
course, a player must also have 1,000 points to begin with, assuming that a game ends with negative
scores. Specific rulesets may handle differently what happens to the bet on a multiple win.

After a riichi declaration, the hand remains locked and unchangeable. In this state, the player is
simply waiting for a winning tile to appear, either by draw or discard. However, a notable exception
may involve calls for kan.

It is not allowed to declare riichi when less than four tiles remain in the wall. The player must be
able to draw one more tile in an uninterrupted set of turns. For any hand, the absolute last chance
to call riichi is when the player has a chance to score riichi, haitei, ippatsu, and mentsumo.

### Noten riichi

A player may actually declare riichi with a noten hand. This is a **noten riichi**
{{< kana "ノテン立直" >}}, which is highly not advisable. Of course, computer based play is usually
programmed to prevent players from making such a riichi declaration. If and when the hand results in
ryuukyoku, then it is required of the player to reveal the hand to show tenpai. The noten riichi
would naturally show noten; and it is subject to chombo. Likewise, chombo is applied if a win is
called, as the hand's winning status would be verified only to reveal an invalid hand. Players may
escape this dubious position, when another player wins the hand or has committed a chombo penalty.
An abortive draw may also prevent the chombo penalty, except suucha riichi.

### Kan

`main|Kan\#Kan during riichi`

Kan during riichi is a discretionary play. Players must take not of the hand composition to
determine, whether the kan is allowable or not. Disallowable calls either alters the hand's waiting
pattern or alters the hand's composition, where tile groupings are changed. Improper calls are
subject to chombo.

### Furiten

`main|Furiten`

During riichi, a player may be furiten. In this case, the riichi is considered to be a **furiten
riichi**. Likewise, after calling riichi, a player may decline the option to call on a winning tile,
whether it be by self-draw or by discard. However, if a win is declined, then the riichi declared
hand is subject to furiten for the remainder of the hand. Thus, all tiles discarded after the riichi
declaration, as indicated by the discarded sideways tile, are considered to be guaranteed safe
tiles. While it is advised to declare a win on the first opportunity, some plays may require a
player not to do so under very specific circumstances. Of course, such play requires caution.

### Uradora

![The benefits of uradora. [1](http://tenhou.net/0/?log=2014070812gm-0009-7447-daa26980&tw=3&ts=9)](Uradora.png "The benefits of uradora. 1")
`main|Dora`

When a riichi declarer wins, he may flip the tiles underneath the dora and kan dora indicators.
These flipped _ura dora indicators_ may increase the value of his hand.

### Tochuu ryuukyoku

`main|Tochuu ryuukyoku`

Tochuu ryuukyoku, or abortive draw, is in effect when suucha riichi occurs. In this case, three
riichis have been called; and a fourth one is called in place. If the discarded tile is not claimed
for a win, then the hand ends.

### Open riichi

**Open riichi** {{< kana "オープン立直" >}} is an optional yaku, which is a modified version to
riichi itself. The principle works exactly in the same way as the original riichi. However, there is
an added bonus of 1-han attached, which is earned by revealing either the hand or the tile waits.
This naturally eases everyone else's ability to defend against the riichi. In general, the intent
involves winning by self-draw. As a further optional rule, yakuman may be awarded if any player
actually plays into the open riichi. This is more or less a harsh penalty for playing into a
revealed hand. Many official organizations and tournaments do not implement open riichi. It is more
or less reserved for casual or gambling game settings.

### Oikake riichi

For any given hand, more than one player may call riichi. Subsequent riichi calls after the first is
called **oikake riichi** {{< kana "追いかけリーチ" >}}. In English, this is often referred to as the
"chase riichi". Oikake has one distinct advantage of an ippatsu ron chance against any previous
riichi hands before taking a draw. After all, defense is no longer a factor to those hands, as they
are committed to discarding non-winning tiles. The hands are essentially engaged in a "riichi duel"
provided that they have different waiting tiles. Under many rulesets, the hand aborts in the event
for all players to have declared riichi via suucha riichi.

### Karaten riichi

Riichi may actually be called if all waiting tiles are rendered unavailable. A hand in this state is
rendered as karaten. In this state, the hand is not winnable as the number of available winning
tiles is zero. Such calls may be made by mistake, or intentionally to cause players to defend.

## Riichi based yaku

Two yaku are specifically associated with riichi: **ippatsu** and **double riichi**.

### Ippatsu

`main|Ippatsu`

**Ippatsu** {{< kana "一発" >}} is a separate yaku but is very dependent on riichi. Riichi is
required to score ippatsu, as the yaku is defined as a riichi winning before the declarer's next
discard. Tile calls, including kan, are able to nullify ippatsu by denying it.

### Double riichi

`main|Double riichi`

**Double riichi** {{< kana "ダブルリーチ" >}} is a special case for riichi. Here, the player
declares riichi on the first turn before one's own discard. No tile call may have interrupted the
turn order before the declaration, where applicable. For the dealer, double riichi can never be
interrupted. As the name implies, this yaku is worth 2 han instead of 1 han, as a bonus for the
initial timing. With the added han value, a call for double riichi has a huge advantage over the
other players in conjunction with the immediate speed and value compared to conventional riichi
calls.

## Compatibility

`main|Yaku compatibility`

`Yaku compatibility table|RCH,DRI`

Given the rules, riichi is compatible with all other yaku. Likewise, riichi and double riichi cannot
be combined as they are essentially the same yaku. It just happens that double riichi is a specific
instance of riichi. Per rule and definition, it is impossible to attain ippatsu without riichi.

Once again, the prospect for rinshan kaihou to work with riichi is dependent on the legality of the
kan call during riichi. With computer interfaces, this rule may be enforced by disabling a player's
ability to call the kan during this situation. However, with actual tiles, a player must be aware of
this legality. Otherwise, chombo may be enforced instead.

## Strategy

`main|Riichi strategy`

Calling riichi is an optional play. Therefore, players often have to make various considerations,
when it comes to using riichi. Often, this depends on the discarded tiles, how early or late in the
hand, the hand value, and/or the player's hand composition. If possible, a player may opt not to use
riichi at all and instead employ the strategy of damaten.

## External links

`jpwiki|立直`

- [Yaku, Hai\!: Riichi (& Ippatsu & Daburu Riichi)](http://osamuko.com/yaku-hai-riichi/)

<!-- end list -->

-


    [Osamuko](http://osamuko.com/) article on riichi.

`Navbox yaku`

- [Source of this page [arcturus wiki]](http://arcturus.su/wiki/Riichi)
